legacy/ui/legacy/src/main/java/com/fsck/k9/ui/settings/general/GeneralSettingsDataStore.kt

```diff
@@ -102,6 +102,22 @@ class GeneralSettingsDataStore(
         saveSettings()
     }
 
+    override fun getLong(key: String, defValue: Long): Long {
+        return when (key) {
+            "fundingReminderShownTimestamp" -> K9.fundingReminderShownTimestamp
+            else -> defValue
+        }
+    }
+
+    override fun putLong(key: String?, value: Long) {
+        when (key) {
+            "fundingReminderShownTimestamp" -> K9.fundingReminderShownTimestamp = value
+            else -> return
+        }
+
+        saveSettings()
+    }
```

Comment:
`GeneralSettingsDataStore` is only used by the settings screen. Since we don't have a visual setting, this is not necessary.

Reply:
It would be good to document how this is supposed to work. From reading the implementation, it's not obvious.